doc/source/admin/framework_dependencies.rst

```diff
@@ -0,0 +1,573 @@
+
+Install dependencies using the `unpinned requirements file`_, and then instruct
+Galaxy to start without attempting to fetch wheels:
+
+.. code-block:: console
+
+    (venv)$ pip install --index-url=https://wheels.galaxyproject.org/simple/ -r lib/galaxy/dependencies/requirements.txt
+    (venv)$ deactivate
+    $ sh run.sh --no-create-venv --skip-wheels
+
+Including ``--index-url=https://wheels.galaxyproject.org/simple/`` is important
+- at least one current Galaxy dependency (SVGFig) is not available in PyPI but
+is available (in both source and wheel form) on `wheels.galaxyproject.org`_,
+and two (pysam, Whoosh) include modifications specific to Galaxy which are only
+available on `wheels.galaxyproject.org`_.
+
+.. _unpinned requirements file: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/requirements.txt
+
+Wheel interaction with other software
+-------------------------------------
+
+Galaxy job handlers
+^^^^^^^^^^^^^^^^^^^
+
+All Galaxy jobs run a metadata detection step on the job outputs upon
+completion of the tool. The metadata detection step requires many of Galaxy's
+dependencies. Because of this, it's necessary to make sure the metadata
+detection step runs in Galaxy's virtualenv. If you run a relatively simple
+Galaxy setup (e.g. single process, or multiple Python Paste processes started
+using ``run.sh``) then this is assured for you automatically. In more
+complicated setups (using a cluster, supervisor, and/or the "headless" Galaxy
+handler) it may be necessary to make sure the handlers know where the
+virtualenv (or a virtualenv containing Galaxy's dependencies) can be found.
+
+If your jobs are failing due to Python ``ImportError`` exceptions, this is most
+likely the problem. If your jobs run on a cluster, you can use the ``<env>``
+tag in ``job_conf.xml`` to source the virtualenv. For example:
+
+.. code-block:: xml
+
+    <job_conf>
+        <plugins>
+            ...
+        </plugins>
+        <destinations default="cluster">
+            <destination id="cluster" runner="drmaa">
+                <param id="nativeSpecification"> ...cluster options... </param>
+
+                <env file="/galaxy/server/.venv/bin/activate" />
+
+            </destination>
+        </destinations>
+    </job_conf>
+
+If your Galaxy server's virtualenv isn't available on the cluster you can
+create one manually using the instructions under `Managing dependencies
+manually`_.
+
+Pulsar
+^^^^^^
+
+If using `Pulsar`_'s option to set metadata on the remote server, the same
+conditions as with `Galaxy job handlers`_ apply. You should create a virtualenv
+on the remote resource, install Galaxy's dependencies in to it, and set an
+``<env>`` tag pointing to the virtualenv's ``activate`` as in the `Galaxy job
+handlers`_ section. Instructions on how to create a virtualenv can be found
+under the `Managing dependencies manually`_ section.
+
+.. _Pulsar: http://pulsar.readthedocs.org/
+
+Conda
```

Comment:
As I understand it Conda is very much experimental feature and thus I feel it belongs to the very end of this document.